doc/whatsnew/fragments/10559.new_check

```diff
@@ -1,6 +1,6 @@
-Add new checks for invalid uses of class patterns in ``match``.
-* ``invalid-match-args-definition`` is emitted if ``__match_args__`` isn't a tuple of strings.
-* ``too-many-positional-sub-patterns`` if there are more positional sub-patterns than specified in ``__match_args__``.
-* ``multiple-class-sub-patterns`` if there are multiple sub-patterns for the same attribute.
+Add new checks for invalid uses of class patterns in :keyword:`match`.
+* :ref:`invalid-match-args-definition` is emitted if :py:data:`object.__match_args__` isn't a tuple of strings.
+* :ref:`too-many-positional-sub-patterns` if there are more positional sub-patterns than specified in :py:data:`object.__match_args__`.
+* :ref:`multiple-class-sub-patterns` if there are multiple sub-patterns for the same attribute.
```

Comment:
Ref #10559

Reply:
Did you see https://github.com/pylint-dev/pylint/pull/10568 ?

Reply:
Yes. It looks quite promising though I only skimmed through it so far.